### PR TITLE
RCDs can build computer & machine frames on tiles containing directional windows

### LINKED ISSUE #59507: RCDs do not like directional windows.
Step one: Construct a directional glass window
Step two: use an RCD to construct a frame or computer (on the tile containing the directional glass window)
Step three: the rcd makes a "click" sound but does nothing, it neither produces the frame, nor errors

I beg, please just make it work instead of making it so you DEFINITELY can't

While we're here, you can't put two windoors on one tile with an RCD, despite the fact that you can. (oh god please make this work instead of removing the ability to do it without the rcd, it's good for making doors like the HOP's)

image

In this example picture, the machine frames needed to be created by hand, or with the RCD in a 100% "free" tile and manually pushed/wrenched in place.

Sorry about this, guy who hates directional glass!

### PR DESCRIPTION
**Pull request by SyncIt21**
## About The Pull Request

If a turf has a directional window or a reinforced directional window it now won't stop, you from making a machine/computer frame on that turf with the RCD

Fixes #59507

## Changelog

:cl:
fix: Directional windows won't block construction of machine & compute frames by the RCD
/:cl:

### CODE REVIEW

**Comment by SyncIt21** [comment]
Merge/Review anyone?

File: code/game/objects/items/RCD.dm [hunk 1]
```diff
@@ -536,7 +536,14 @@ GLOBAL_VAR_INIT(icon_holographic_window, init_holographic_window())
 		return FALSE
 	if(rcd_results["mode"] == RCD_MACHINE || rcd_results["mode"] == RCD_COMPUTER || rcd_results["mode"] == RCD_FURNISHING)
 		var/turf/target_turf = get_turf(A)
-		if(target_turf.is_blocked_turf(exclude_mobs = TRUE))
+		//ignore all directional windows on the turf
+		var/static/list/ignore_types = list(/obj/structure/window, /obj/structure/window/reinforced)
+		var/list/directional_windows = list()
```
Comment (Time-Green) [inline code/game/objects/items/RCD.dm:541]: Can you name this something more generic? Like ignored atoms

**Review by Time-Green — CHANGES_REQUESTED** [review]

**Comment by SyncIt21** [comment]
@Time-Green good to see you :)

**Comment by Time-Green** [comment]
> @Time-Green good to see you :)

:)

**Pull request merged**